bin/deploy-intensive-worker.sh

```diff
@@ -10,7 +10,7 @@ then
   echo "Using AWS_PROFILE=${AWS_PROFILE}";
   echo "      AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}";
 
-  if [[ ! -f ${1}-worker-params.yml ]]
+  if [[ ! -f ${1}-ecs-params.yml ]]
```

Comment:
last I checked, get-params doesn't make -ecs-params.yml anymore.

Reply:
this might be a problem, since many of the scripts still depend on it existing.